ibrcommon/tests/ssl/CipherStreamTest.cpp

```diff
@@ -299,13 +301,14 @@ void CipherStreamTest::aesstream_test01()
 
 	// decrypt the test data
 	{
-		ibrcommon::AES128Stream crypt_stream(ibrcommon::CipherStream::CIPHER_DECRYPT, data, key, salt, iv);
+		refcnt_ptr<ibrcommon::AES128Stream> crypt_stream = refcnt_ptr<ibrcommon::AES128Stream>(
+				new ibrcommon::AES128Stream(ibrcommon::CipherStream::CIPHER_DECRYPT, data, key, salt, iv));
```

Comment:
Same as above. IMHO simple initialization would be better.

Reply:
Done.